apps/calendar/locales/calendar.en-US.properties

```diff
@@ -44,7 +44,11 @@ advanced-settings-add-account=Add Account
 field-user.placeholder=Username
 field-password.placeholder=Password
 field-full-url.placeholder=URL
-modify-account-setting-up=Please wait while your account is set up
+<<<<<<< HEAD
```

Comment:
The new string looks fine, but you need to remove the diff markers before we can land it. Thanks.